drivers/timer/nrf_rtc_timer.c

```diff
@@ -186,18 +187,43 @@ void rtc_nrf_isr(void *arg)
 	z_clock_announce(IS_ENABLED(CONFIG_TICKLESS_KERNEL) ? dticks : (dticks > 0));
 }
 
-int z_clock_driver_init(struct device *device)
+static void clock_on_notify(struct onoff_manager *mgr,
```

Comment:
Most of the changes introduced here are removed in commit f2d1f8297f15bc4cffcb78bf7fd92700e339d7df (drivers: timer: nrf: Modify LF clock request). Maybe it would be better to change this file in one step?

Reply:
done